hooks/useProposalVotes.tsx

```diff
@@ -18,23 +24,31 @@ export default function useProposalVotes(proposal?: Proposal) {
   // TODO: optimize using memo
   if (!realm || !proposal || !governance || !proposalMint)
     return {
-      voteThresholdPct: 100,
```

Comment:
in my opinion hooks should not obfuscate whether data is loading or just an optimistic default, so here i made the components individually decide how to handle loading data.